Fishing Game/ViewControllers/FishingScreenViewController.swift

```diff
@@ -38,11 +43,45 @@ class FishingScreenViewController: UIViewController {
     @IBOutlet weak var distanceToFishLabel: UILabel!
     @IBOutlet weak var timeRemainingLabel: UILabel!
     @IBOutlet weak var baitRemainingLabel: UILabel!
+    @IBOutlet weak var baitImage: UIImageView!
+    
+    @IBOutlet weak var baitViewsHStack: UIStackView!
+    @IBOutlet weak var fishingProgressView: UIImageView!
+    @IBOutlet weak var fishRadarView: UIImageView!
     
     // Displays your progress in the game
     @IBOutlet var clockLabel: UILabel!
     
     override func viewDidLoad() {
+        fishingProgressView.layer.cornerRadius = 10
+        fishRadarView.layer.cornerRadius = 10
+        baitImage.layer.borderColor = UIColor.systemYellow.cgColor
+        baitImage.layer.borderWidth = 2
+        baitImage.layer.cornerRadius = baitImage.frame.height / 2
```

Comment:
Some of these you can do in storyboard. I usually like to do everything in the storyboard that is possible. and then use the view controller to do anything you can't do in storyboard.